File: appkit-creating-and-customizing-the-touch-bar/Objective-C/NSTouchBar Catalog/SplitViewController.m
/*
See the LICENSE.txt file for this sample’s licensing information.

Abstract:
Main split-view controller for this sample (primary = table of tests, detail = each view controller test).
*/

#import "SplitViewController.h"

@implementation SplitViewController

- (void)viewDidLoad
{
    [super viewDidLoad];

    self.splitView.autosaveName = @"SplitViewAutosSave";
    
    self.minimumThicknessForInlineSidebars = 10.0;
}

@end

File: appkit-creating-and-customizing-the-touch-bar/Swift/NSTouchBar Catalog/SplitViewController.swift
/*
See the LICENSE.txt file for this sample’s licensing information.

Abstract:
Main split-view controller for this sample (primary = table of tests, detail = each view controller test).
*/

import Cocoa

class SplitViewController: NSSplitViewController {

    override func viewDidLoad() {
        super.viewDidLoad()
        
        splitView.autosaveName = "SplitViewAutosSave"
        minimumThicknessForInlineSidebars = 10.0
    }
    
}

